pages/api/stories/[language].ts

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs';
+import { NextApiRequest, NextApiResponse } from 'next';
+import path from 'path';
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
Why do we need a separate endpoint for this? Feels like you should be able to just directly import a JSON with all stories for the story listing and then directly import the JSON with one story for each story page.

Reply:
Thx, as discussed, I will change to use dynamic import instead

Reply:
reworked